include/jwt.h

```diff
@@ -572,6 +572,20 @@ JWT_EXPORT int jwt_dump_fp(jwt_t *jwt, FILE *fp, int pretty);
  */
 JWT_EXPORT char *jwt_dump_str(jwt_t *jwt, int pretty);
 
+/**
+ * Return plain text representation of grants as a string.
+ *
+ * Similar to jwt_dump_str() except that a grants string is returned. The string
```

Comment:
I would say something more like "except that only a string containing the grants is returned" to be more clear.